content/en/logs/explorer/search_syntax.md

```diff
@@ -49,32 +49,24 @@ Use the syntax `*:search_term` to perform a full-text search across all log attr
 
 ### Single term example
 
-| Search syntax | Search type | Description                                           |
-| ------------- | ----------- | ----------------------------------------------------- |
-| `*:hello` | Full-text   | Searches all log attributes for the term `hello`.     |
-| `hello`       | Free text   | Searches only the log message for the term `hello`.   |
+| Search syntax | Search type | Description                                               |
+| ------------- | ----------- | --------------------------------------------------------- |
+| `*:hello`     | Full-text   | Searches all log attributes for the exact string `hello`. |
+| `hello`       | Free text   | Searches only the log message for the word `hello`.       |
 
 ### Search term with wildcard example
 
-| Search syntax | Search type | Description                                                                                  |
-| ------------- | ----------- | -------------------------------------------------------------------------------------------- |
-| `*:hello` | Full-text   | Searches all log attributes for the exact string `hello`.                                    |
-| `*:hello*`| Full-text   | Searches all log attributes for strings with `hello`. For example, `hello_world`.|
+| Search syntax | Search type | Description                                                                                 |
+| ------------- | ----------- | ------------------------------------------------------------------------------------------- |
+| `*:hello`     | Full-text   | Searches all log attributes for the exact string `hello`.                                   |
+| `*:hello*`    | Full-text   | Searches all log attributes for strings starting with `hello`. For example, `hello_world`.  |
```

Comment:
I found a ticket (https://datadoghq.atlassian.net/browse/DOCS-9144) that removed `starting` from this description after customer reported it's matching all logs with `hello` word inside it. I tested it on org 2 and it's not the case. I think customer got confused in some way, but they report was not valid.